UCSLibraries/src/main/java/it/cnr/iit/utility/errorhandling/Reject.java

```diff
@@ -270,7 +270,7 @@ public static void ifNull( Object... objects ) {
         }
     }
 
-    public static void ifNull( String... objects ) {
+    public static void ifNullStringArray( String... objects ) {
```

Comment:
why is this necessary? I mean a method for type String, isnt the existing one recognise the String as object

Reply:
yes this can be removed